src/open_inwoner/templates/pages/plans/file_upload_form.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n solo_tags form_tags button_tags icon_tags %}
+
+{% get_solo 'openzaak.OpenZaakConfig' as openzaak_config %}
+
+<div id="form_upload">
+{% render_form id="document-upload" form=form method="POST" form_action=form_action enctype="multipart/form-data" submit_text=_("Bestand uploaden") extra_classes="file-input-form" %}
+    {% csrf_token %}
+    {# Name of the file is a hidden input #}
+    {% file_input form.file multiple=False primary_text=_("Upload document") max_upload_size=openzaak_config.max_upload_size allowed_file_extensions=openzaak_config.allowed_file_extensions %}
```

Comment:
`primary_text=_("Upload document")` is the default on the form tag, so not necessary here (removing it here means we won't have to stop by here if we decide to change the text)